ast/format_test.go

```diff
@@ -0,0 +1,222 @@
+package ast_test
+
+import (
+	"testing"
+
+	"github.com/influxdata/flux/ast"
+	"github.com/influxdata/flux/parser"
+	"github.com/pkg/errors"
+)
+
+var skip = map[string]string{
+	"array_expr":           "without pars -> bad syntax, with pars formatting removes them",
```

Comment:
The `()` should not be needed.

Reply:
Rechecked, It returns `bad syntax` without parenthesis.